docs/source/notebooks/mmm/mmm_example.ipynb

```diff
@@ -20,25 +20,101 @@
     "  - We also know that there is a yearly seasonality effect.\n",
```

Comment:
<p>I don't like "causally compounded" because it sounds like "confounded". Maybe reword to emphasise "causal data generating process"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/pymc-marketing/pull/1003/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Rephrase to `causally related.` sounds good? @drbenvincent

Reply:
<p>Add explicitly what DAG means, i.e.</p><p> </p><p>"we can draw a directed acyclic graph (DAG) or a graphical model"</p>